Loop/Managers/DeviceDataManager.swift

```diff
@@ -67,6 +67,42 @@ final class DeviceDataManager {
             UserDefaults.appGroup?.pumpManagerRawValue = pumpManager?.rawValue
         }
     }
+    
+    // MARK: Stores
+    let healthStore: HKHealthStore
```

Comment:
Moving authorization stuff out of LDM and into device data manager per @ps2's suggestion